packages/access-client/test/capabilities/upload.test.js

```diff
@@ -328,6 +328,7 @@ describe('upload capabilities', function () {
       audience: w3,
       with: account.did(),
       proofs: [await any],
+      nb: {},
```

Comment:
Types require empty object when all nb properties are optional. need to look

Reply:
this should not happen theres a lot of infer types in ucanto to mitigate this. Did you open an issue in ucanto ?